Dockerfile

```diff
@@ -17,8 +15,8 @@ RUN if ! rpm -q libvirt-devel; then yum install -y libvirt-devel && yum clean al
 COPY --from=builder /go/src/github.com/openshift/hive/bin/manager /opt/services/
 COPY --from=builder /go/src/github.com/openshift/hive/bin/hiveadmission /opt/services/
 COPY --from=builder /go/src/github.com/openshift/hive/bin/hiveutil /usr/bin
-COPY --from=builder /go/src/github.com/openshift/hive/bin/hive-operator /opt/services
-COPY --from=builder /go/src/github.com/openshift/hive/bin/hive-apiserver /opt/services/
+COPY --from=builder /go/src/github.com/openshift/hive/bin/operator /opt/services/hive-operator
+#COPY --from=builder /go/src/github.com/openshift/hive/bin/hive-apiserver /opt/services/
```

Comment:
no need to keep the old copy command around

Reply:
This is an oversight where I failed to put back the code for making the hiveapiserver. This line does need to be uncommented and kept--along with more work in the makefile.

Reply:
This has been resolved now. I created a Makefile in v1alpha1apiserver to build the apiserver. The main Makefile will call into the apiserver Makefile for build.

I did not add generate and test code to the apiserver Makefile. I did not think it was worth the effort since (1) the apiserver code should not be changing and (2) the apiserver code will be removed soon.